src/main/java/seedu/address/logic/filters/Filters.java

```diff
@@ -21,23 +21,27 @@ public static AbstractFilter getCorrespondingFilter(String info) {
         ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(" " + info + " ", PREFIX_NAME,
             PREFIX_EMAIL,
             PREFIX_PHONE,
-            PREFIX_ADDRESS, PREFIX_DOB, PREFIX_TAG, PREFIX_CARS_OWNED, PREFIX_CARS_PREFERRED);
+            PREFIX_ADDRESS, PREFIX_DOB, PREFIX_TAG, PREFIX_CARS_OWNED, PREFIX_CARS_PREFERRED, PREFIX_COE_EXPIRY);
 
 
-        if (argumentMultimap.getTotalSize() != 2) { // since there is also a dummy position :(
+        if (argumentMultimap.getTotalSize() > 2) { // since there is also a dummy position :(
+            System.out.println(argumentMultimap);
             throw new NullPointerException(
                 "Number of filters between two logical operators should be exactly 1 " + argumentMultimap);
         }
 
         if (argumentMultimap.getValue(PREFIX_EMAIL).isPresent()) {
+            System.out.println(100);
             return new EmailFilter(argumentMultimap.getValue(PREFIX_EMAIL).get());
         }
 
         if (argumentMultimap.getValue(PREFIX_PHONE).isPresent()) {
+            System.out.println(100);
             return new PhoneNumberFilter(argumentMultimap.getValue(PREFIX_PHONE).get());
         }
 
         if (argumentMultimap.getValue(PREFIX_COE_EXPIRY).isPresent()) {
+            System.out.println(100);
```

Comment:
Are these debug code left over?

Reply:
hey, my bad, have removed the debug codes.